addons/l10n_din5008/static/src/scss/report_din5008.scss

```
@@ -6,11 +6,11 @@
     &.header {
         table {
             width: 100%;
-            img, h3, td, tr {
+            img, div.h3, td, tr {
```

```suggestion
            img, h3, div.h3, td, tr {
```
I think we should keep style for H3.

It's possible that some people had a tagline and no issue (eg. if they didn't have a `<p/>` tag in their report_header) and for them the current code should continue to work even if they don't do update `l10n_din5008`, the style should still be working if they still have `<h3/>`.